spec/requests/v1/complexities_request_spec.rb

```diff
@@ -24,21 +24,29 @@
 
 RSpec.describe "Complexities", type: :request do
   let(:response_json) { JSON.parse(response.body) }
+
   let(:request_headers) do
-    # Include an Authorization header to make the request valid
-    { "Authorization" => auth_header }
+    { "Authorization" => "Bearer BOBBINS" }
   end
+
   let(:topic) { instance_double("topic", publish: nil) }
+  let(:expired_token) { false }
 
   before do
     allow(ComplexityEventService).to receive(:sns_topic).and_return(topic)
+
+    stub_access_token(
+      scopes: %w[read write], 
+      roles: [ApplicationController::READ_ROLE, ApplicationController::WRITE_ROLE],
+      expired_token: expired_token
+    )
   end
 
   describe "GET /v1/complexity-of-need/offender-no/:offender_no" do
     let(:endpoint) { "/v1/complexity-of-need/offender-no/#{offender_no}" }
     let(:offender_no) { complexity.offender_no }
     let!(:complexity) { create(:complexity) }
-
+    
```

Comment:
_Extremely_ minor thing of unnecessary whitespace. If you have to go back in to change anything else and remember that it's there then it may be worth deleting, otherwise it's no biggie!

Reply:
@doctorpod can you configure your editor to delete extraneous whitespace from the end of lines and end of files?